astropy/coordinates/baseframe.py

```diff
@@ -146,7 +146,8 @@ def __new__(mcls, name, bases, members):
 
             if (not found_repr_info and
                     '_frame_specific_representation_info' in m):
-                repr_info = m['_frame_specific_representation_info']
+                # create a copy of the dict so we don't mess with the contents
+                repr_info = dict(m['_frame_specific_representation_info'])
```

Comment:
Just `m['_frame_specific_representation_info'].copy()`?

Reply:
FWIW, I just tested this out of idle curiosity, and it seems they are basically identical in speed. (Stylistically, I have no opinion beyond "I tend to use copy because it makes more sense to me".)

Reply:
I switched it to `.copy()`